frontend/src/metabase/visualizations/components/ClickActions/ClickActionsPopover.tsx

```diff
@@ -169,6 +169,8 @@ export class ClickActionsPopover extends Component<
               name: "preventOverflow",
               options: {
                 padding: 16,
+                altAxis: true,
+                tether: false,
```

Comment:
Can we add a repro test for the bug?

We could assert that `document.body.parentElement.scrollHeight` is not greater than `document.body.parentElement.offsetHeight`.

Reply:
I've added the test, but had to use a different element to do the check on.